app/scripts/motions.directive.js

```diff
@@ -13,50 +13,18 @@
 angular.module('dashboard')
     .directive('dbMotions', [function () {
 
-        var controller = ['$log', 'AhjoMeetingSrv', 'StorageSrv', 'CONST', '$rootScope', function ($log, AhjoMeetingSrv, StorageSrv, CONST, $rootScope) {
+        var controller = ['$log', 'StorageSrv', 'CONST', '$rootScope', '$scope', function ($log, StorageSrv, CONST, $rootScope, $scope) {
             $log.log("dbMotions: CONTROLLER");
             var self = this;
             self.loading = false;
-            self.motions = null;
+            self.motions = StorageSrv.getKey(CONST.KEY.MOTION_ARRAY);
```

Comment:
StorageSrv.getKey may return `null`. Please consider should `StorageSrv` be updated to return `[]` or a check added here.